scripts/gen-api-gtkdoc.py

```diff
@@ -396,13 +399,25 @@ def main(args):
     else:
         outfile = sys.stdout
 
+    if (opts.scioutfile):
+        try:
+            scioutfile = open(opts.scioutfile, "w+")
+        except OSError as err:
+            sys.stderr.write("failed to open \"%s\" for writing (%s)\n" % (opts.scioutfile, err.strerror))
+            return 1
+    else:
+        scioutfile = outfile
+
     try:
         outfile.write("/*\n * Automatically generated file - do not edit\n */\n\n")
-        outfile.write("#include <glib.h>\n")
-        outfile.write("#include <gtk/gtk.h>\n\n")
-        outfile.write("typedef struct _ScintillaObject ScintillaObject;\n")
+        outfile.write("#include \"gtkcompat.h\"\n")
+        outfile.write("#include \"sciwrappers.h\"\n")
```

Comment:
Oh, and I'm not sure about "" vs <>. I doubt it makes a difference, as IIUC g-ir-scanner doesn't have default or system include directories but only those you mention in CFLAGS.

EXCEPT for the introspection program that is generated and executed. here the CFLAGS are passed to the compiler. The introspection program uses the get-type.txt and intercepts gobject type registration including signal and property registration.